index.html

```diff
@@ -4258,10 +4258,27 @@ <h1>DID Methods</h1>
             <a href="https://workday.github.io/work-did-method-spec/">Workday DID Method</a>
           </td>
         </tr>
+        <tr>
+          <td>
+            did:polygon:
```

Comment:
Methods are in sorted order.  Please move this to proper position.

Reply:
Hey, thanks for your comments. As per it we have rearranged the index.html file in alphabetically sorted format. Thank you.